users.view.lkml

```diff
@@ -74,13 +74,23 @@ view: users {
     sql: ${TABLE}.user_fields__team_lead ;;
   }
 
-  dimension: bpo_cohort {
-    label: "BPO Cohort"
-    description: "Cohort identifier for third-party service provider. This field will be a combination of the agent's start date and starting location."
-    type: string
-    sql: ${TABLE}.user_fields__bpo_start_date ;;
+  dimension_group: time_bpo_cohort_start_at {
+    description: "Week that the third-party service provider started. This field is used to define BPO cohorts"
```

Comment:
May I suggest adding a note in description, mentioning this is user input and doesn't have a specific timezone?

Reply:
Updated - thanks for the suggestion.  I agree that this is more clear.